utils/report.c

```diff
@@ -1105,6 +1123,7 @@ TEST_CASE(report_sort)
 		fstack[i].addr = i;
 		fstack[i].total_time = total_times[i];
 		fstack[i].child_time = child_times[i];
+		rstack[i].depth = depth[i];
 	}
 
 	for (i = 0; i < TEST_NODES; i++) {
```

Comment:
Could you please add explicit checks of min/max depth to the test case?

Reply:
I added checking depth min/max in `report_sort` and `report_diff` unit tests.